### PR TITLE
Fix Custom Networks for 'enable' calls

### PR DESCRIPTION
**Pull request by janmarcano**

### CODE REVIEW

File: packages/extension/src/background/transaction/actions.ts [hunk 1]
```diff
@@ -160,17 +160,12 @@ export function getLedgerFromMixedGenesis(genesisId: string, genesisHash: string
...
     const injectedNetworks = Settings.getCleansedInjectedNetworks();
-    injectedNetworks.forEach(network => {
-      if (network['genesisId'] === genesisId) {
-        // Found genesisId, make sure the hash matches 
-        if (!genesisHash || genesisHash === network.genesisHash) { 
-          return network;
-        }
-      }
-    });
+    ledger = injectedNetworks.find((network) => network['genesisId'] === genesisId);
```
Comment (PureBrent) [inline packages/extension/src/background/transaction/actions.ts:165]: The ARCs requires the ID and the hash both to match if the match is provided.  So should not return if the hash is different and should continue the checks to allow for a duplicate genesisId with a different hash.

File: packages/extension/src/background/transaction/actions.ts [hunk 2]
```diff
@@ -183,17 +178,7 @@ export function getLedgerFromMixedGenesis(genesisId: string, genesisHash: string
...
-    const injectedNetworks = Settings.getCleansedInjectedNetworks();
-    injectedNetworks.forEach(network => {
-      if (network['genesisHash'] === genesisHash) {
-        // Found genesisHash, make sure the id matches 
-        if (!genesisId || genesisId === ledger.genesisId) { 
-          return network;
-        }
-      }
-    });
+    // We don't currently store the genesisHash of the custom networks
```
Comment (PureBrent) [inline packages/extension/src/background/transaction/actions.ts:181]: Should we not still check hash here and add the storage capability on the custom networks page?

**Review by PureBrent — CHANGES_REQUESTED** [review]

File: packages/extension/src/background/transaction/actions.ts (hunk 2 above)
Comment (janmarcano) [reply packages/extension/src/background/transaction/actions.ts:181]: This is a temporary workaround until we add support for storing hashes on custom networks in a future release.

File: packages/extension/src/background/transaction/actions.ts (hunk 1 above)
Comment (janmarcano) [reply packages/extension/src/background/transaction/actions.ts:165]: same as below's comment

**Review by PureBrent — APPROVED** [review]

**Pull request merged**